src/gtest/test_joinsplit.cpp

```diff
@@ -547,3 +547,25 @@ TEST(joinsplit, note_plaintexts)
 
     ASSERT_TRUE(decrypted.memo == note_pt.memo);
 }
+
+TEST(joinsplit, note_class)
```

Comment:
`TEST(joinsplit, NoteClass)` (no underscores in Google Test testcase names - something we need to clean up at some stage, but we shouldn't introduce more)